packages/cli/src/bin/index.ts

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env ts-node
+import semver from 'semver';
+import chalk from 'chalk';
+import program from 'commander';
+import { init } from '../lib/init';
+import { log } from '../lib/log';
+import { board } from '../lib/board';
+import { start } from '../lib/start';
+import { devPlugin } from '../lib/devPlugin';
+import { dataset } from '../lib/dataset';
+import { serve } from '../lib/serve';
```

Comment:
How to export the above modules as default mode, just like:
```suggestion
export default const dataset: CommandHandler = async ({ type }) => {
```

So that we can import these:

```js
import init from '../lib/init';
import log from '../lib/log';
...
```
And how about moving these command handlers into a separate directory, such as "commands".

Reply:
IMO it is unnecessary, bin/index.ts is a polymerization of all command and have not other code here :(  It is unnecessary to add a new dir for 'commands'

Reply:
The "lib" is a generic name, we should have more specific name here, BTW the src/lib is weird a bit, alternatively you could also move these lib files into "src".

Reply:
Resolved, how about this.